railgun/internal/ctrlutils/vars.go

```diff
@@ -6,3 +6,4 @@ package ctrlutils
 
 // KongIngressFinalizer is the finalizer used to ensure Kong configuration cleanup for deleted resources.
 const KongIngressFinalizer = "configuration.konghq.com/ingress"
+const KnativeIngressFinalizer = "networking.internal.knative.dev/ingress"
```

Comment:
Can you help me to understand the impetus to add this finalizer? Generally speaking when a controller manages objects that belong to another API group (which themselves have their own controllers with finalizers) you would simply use a finalizer unique to the controller at hand such as the existing one. What led to adding this new one?

Reply:
That was added to debugging message flow between two controller. No kong code use it. remove it now.